tests/web/test_info.py

```diff
@@ -0,0 +1,5 @@
+def test_api_endpoint_shows_basic_info(client):
+    from bigchaindb import version
```

Comment:
@diminator, that still doesn't explain the reasoning, in `test_transaction.py` the imports are done over and over again, but the modules that are imported are not re-initialized each time (I just tested this), so I'm not sure what the reasoning behind it is.